guide/src/format/configuration/renderers.md

```diff
@@ -279,6 +279,16 @@ The value can be any valid URI the browser should navigate to (e.g. `https://rus
 This will generate an HTML page which will automatically redirect to the given location.
 Note that the source location does not support `#` anchor redirects.
 
+### `[.mdbookignore]`
```

Comment:
I'm not sure why this is in square brackets. The other sections are like that because they are describing TOML sections of `book.toml`.

```suggestion
### `.mdbookignore`
```

Reply:
Fixed.